easybuild/framework/easyconfig/easyconfig.py

```diff
@@ -1226,6 +1226,13 @@ def det_modpath_extensions(self, ec):
         self.log.debug("Obtained modulepath extensions: %s" % modpath_extensions)
         return modpath_extensions
 
+    def det_user_modpath_extensions(self, ec):
+        """Determine user-specific modulepath extensions according to module naming scheme."""
+        self.log.debug("Determining user modulepath extensions for %s" % ec)
+        modpath_extensions = self.mns.det_user_modpath_extensions(self.check_ec_type(ec))
+        self.log.debug("Obtained user modulepath extensions: %s" % modpath_extensions)
```

Comment:
Copy/paste engineering ;-)  The `"..." % foo` version is used all over the place in this file, in particular in the methods surrounding this new one.  I'll change the two instances here but leave the others untouched.  I guess this should be addressed in a separate PR.